src/Core/src/Handlers/Entry/EntryHandler.iOS.cs

```diff
@@ -122,9 +123,7 @@ public static void MapFormatting(IEntryHandler handler, IEntry entry)
 
 		protected virtual bool OnShouldReturn(UITextField view)
 		{
-			view.ResignFirstResponder();
-
-			// TODO: Focus next View
+			KeyboardAutoManager.GoToNextResponderOrResign(view);
```

Comment:
Does it make sense to also add this to `EntryCellRenderer` ? 

![image](https://user-images.githubusercontent.com/5375137/209881393-9738bce1-d242-44c0-8561-7613ba81af94.png)

Reply:
Okay added with some modification and seems to be working well! I'll post a new video at the top of the PR with a tableview